### ISSUE TITLE
awscli-local doesn't read a default region from ~/.aws/config

### ISSUE THREAD

**Issue by kai2nenobu** [localstack/awscli-local#71]
`awslocal` command doesn't read a default region from `~/.aws/config`, but `cdklocal` does. So [an example for LocalStack integration with CDK](https://docs.localstack.cloud/user-guide/integrations/aws-cdk/) doesn't seem to work at first glance.

For example, I configure a default region in `~/.aws/config` as below.

```ini
[default]
region = us-east-2
```

Then, `cdklocal deploy` deploys a stack in `us-east-2` region.

```console
$ cdklocal bootstrap && cdklocal deploy
...
✨  Deployment time: 5.3s

Stack ARN:
arn:aws:cloudformation:us-east-2:000000000000:stack/CdkSampleStack/994c9309
```

A SNS topic has been created in the stack but `awslocal` doesn't show it because `awslocal` investigate `us-east-1` region by default.

```console
$ awslocal sns list-topics
{
    "Topics": []
}
```

If I specify the region with `awslocal`, the SNS topic is visible.

```console
$ awslocal sns list-topics --region us-east-2
{
    "Topics": [
        {
            "TopicArn": "arn:aws:sns:us-east-2:000000000000:CdkSampleStack-CdkSampleTopic2C761410-c0e07754"
        }
    ]
}
```

Wouldn't it be more user-friendly if the behavior of `awslocal` and `cdklocal` were the same?

## Version info

- AWS CLI: 2.9.11 (Python 3.9.11)
- awscli-local: 0.20
- aws-cdk: 2.64.0 (Node.js 18.10.0)
- aws-cdk-local: 2.16.0

**Comment by QAnders** [comment]
I too had this issue and totally tripped me up while using Localstack... I appreciate the effort, but as the new MacBook Mx laptops only comes with Python3 installed and pip3 and trying to get python/pip v2 installed is a real struggle.

We've move to use a bit of a simplified approach, as we use a mix of Mac's and Windows WSL2 (Ubuntu) and have now set this as a standard:

```shell
# .bachrc

# Your nifty aliases goes here:
alias awsl='aws --endpoint-url=http://localhost:4566 '

# AWS PROFILES
awslocalstack() {
  export AWS_REGION=eu-west-1
  export AWS_ACCESS_KEY_ID=test
  export AWS_SECRET_ACCESS_KEY=test
  echo "AWS Profile set to LOCAL"
}
awstest() {
  export AWS_REGION=eu-west-1
  export AWS_ACCESS_KEY_ID=AKIAxxxxxxxxxxxxxx
  export AWS_SECRET_ACCESS_KEY=0xxxxxxxxxxxxxxxx
  echo "AWS Profile set to LOCAL"
}

# LS Key
export LOCALSTACK_API_KEY=xxxxxxxx

# Start working locally at new session
awslocalstack
```

Now we can simply run `awsl rds-data execute-statement ....`

Sure, we miss some of the nifty setting of parameters from `awslocal` but far fewer issues with dependencies and `pip`...

**Comment by viren-nadkarni** [comment]
Hi @kai2nenobu, we have fixed this in [awscli-local 0.21](https://pypi.org/project/awscli-local/0.21/).

Thanks for reporting!
